aws/resource_aws_api_gateway_gateway_response.go

```diff
@@ -0,0 +1,152 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayGatewayResponse() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayGatewayResponseCreate,
+		Read:   resourceAwsApiGatewayGatewayResponseRead,
+		Update: resourceAwsApiGatewayGatewayResponseCreate,
+		Delete: resourceAwsApiGatewayGatewayResponseDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
```

Comment:
Is there any particular reason to have the customizable timeout for this resource?
This feature was IMO mainly aimed at resources which are going to take varying time to create/update/delete depending on fields, e.g. instance size, cluster size, ...

I described the full reasoning here https://github.com/terraform-providers/terraform-provider-aws/pull/945#issuecomment-312916643 (beginning from the 2nd paragraph).